app/Providers/HealthServiceProvider.php

```diff
@@ -76,7 +77,7 @@ public function boot(): void
                 ->unless($local || app()->environment('staging')),
             // cloudflare dns
             PingCheck::new()
-                ->url('1.1.1.1')
+                ->url(str_replace($secure, '', config('app.url')))
```

Comment:
Rather than checking `app.force_https` and doing this string replace you could just get the URL without the schema by grabbing it's parts out of https://www.php.net/manual/en/function.parse-url.php